WaterSmartMeter/ngsild-payloads/smartMeter.json

```diff
@@ -0,0 +1,59 @@
+{
+    "id": "urn:ngsi-ld:Device:smartMeter01",
+    "type": "Device",
+    "name": {
+        "type": "Property",
+        "value": "smartMeter0121215"
+    },
+    "serialNumber": {
+        "type": "Property",
+        "value": "X214356"
+    },
+    "installedAt": {
+        "type": "Property",
+        "value": "2020-03-11T11:00:00Z"
+    },
+    "dateLastCalibration": {
+        "type": "Property",
+        "value": "2020-06-11T11:00:00Z"
+    },
+    "controlledProperty": {
+        "type": "Property",
+        "value": [
+            "waterConsumption",
+            "minFlow",
+            "maxFlow",
+            "persistenceFlowDuration",
+            "acquisitionStageFailure",
+            "alarmFlowPersistence",
+            "alarmInProgressAlarms",
```

Comment:
it should be modified to alarmInProgress

Reply:
done